src/Page/Notification/Notification.tsx

```diff
@@ -114,6 +116,15 @@ export const Notification:React.FC = () => {
       case 'session_request': return <WalletConnectInit payload={eventPayload} closeWindow={closeWindow} />
       case 'provenance_sendTransaction': // fallthrough
       case 'provenance_sign': return <SignRequest payload={eventPayload} closeWindow={closeWindow} />;
+      // Upon successful connection, return success message
+      case 'connect': 
+        return (
+          <Success 
+            title='Wallet Connected!' 
+            subTitle='You are now connected to your Provenance Wallet' 
```

Comment:
Should be something like "You're now connected to [insert dApp name pulled from eventPayload]" if it exists, or else, "You're now connected" might suffice.  The user is always connected to their wallet because this is the wallet app.